### ISSUE TITLE
Add contribution & bug reporting guidelines

### ISSUE THREAD

**Issue by cTxplorer** [rahulkumaran/merkalysis#36]
**Describe the bug**
Contribution guidelines is missing in the repo. Moreover, hyperlinks in [README.md/Contributing](https://github.com/rahulkumaran/merkalysis/blob/master/README.md#contributing) are broken.

**To Reproduce**
1. Go to '[README.md/Contributing](https://github.com/rahulkumaran/merkalysis)'
2. Click on 'contribution guidelines' or 'reporting bugs'

**Expected behavior**
Those links should lead to respective docs in the repo. However, those docs needs to be created before they could be linked.

**Desktop**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 70.0.3538.110

**Comment by rahulkumaran** [comment]
@cTxplorer you can go ahead with this.

**Comment by cTxplorer** [comment]
@rahulkumaran Due to ongoing exams, I will start to work on this in couple of days. Hope, it's okay.

**Comment by rahulkumaran** [comment]
Yeah sure. Shouldn't be a problem. Anyway, the official coding period for KWoC starts only from 10th. So yeah. But thereafter, depending on the difficulty of the issue, the deadline would be fixed.

**Comment by nightcrawler321** [comment]
I have found this project interesting and want to contribute in this .
Please suggest some initial steps , where I should start with.
